package.json

```
@@ -21,6 +21,7 @@
     "postinstall": "patch-package && (is-ci || husky install .husky)",
     "check-all": "npm-run-all prettier lint:fix tsc rss",
     "rss": "node scripts/generateRss.js",
+    "reset": "rm -rf node_modules/.cache && rm -rf .next && yarn cache clean",
```

좀 더 명확한 명령어를 사용하면 어떨까요?
```suggestion
    "cache-reset": "rm -rf node_modules/.cache && rm -rf .next && yarn cache clean",
```

---

넵! `README.md` 내용까지 함께 수정하여 저녁에 커밋 올리도록 하겠습니다🙇‍♂️

---

수정 완료하였습니다👍